src/lib/schematics/utils/package-json.ts

```diff
@@ -8,19 +8,24 @@
 
 import {Tree} from '@angular-devkit/schematics';
 
+/** Function that sorts the keys of the specified object and returns the result as a new object. */
+const sortObjectByKeys = (obj: any) => Object.keys(obj).sort()
+    .reduce((result, key) => (result[key] = obj[key]) && result, {});
```

Comment:
* Define this using the `function` keyword
* I think you can use the `object` type here
* Function description should start with the verb:
```ts
/**
 * Sorts the keys of the given object.
 * @returns A new object instance with sorted keys
 */
```

Reply:
Done. I did it like that because I wanted to somehow make clear that this isn't really part of the `package-json` functions. But maybe not having the `export` modifier is already clear enough.